pkg/container/writer.go

```diff
@@ -120,3 +120,20 @@ func (ctn Writer) toWriter(header header, w io.Writer) (err error) {
 
 	return ipld.EncodeStreaming(encoder, node, cbor.Encode)
 }
+
+// ToReader convert a container Writer into a Reader.
+// Most likely, you only want to use this in tests for convenience.
```

Comment:
> Most likely

Since it's not a test package, shouldn't we just return `(io.Reader, error)` and let the user decide?